wave_vision/include/wave/vision/fast_detector.hpp

```diff
@@ -94,9 +98,9 @@ class FASTDetector : public FeatureDetector {
     *  @param check_config, the FASTParams struct containing the desired
     *  configuration values.
     */
-    void checkConfiguration(FASTParams& check_config);
+    void checkConfiguration(const FASTParams& check_config);
 };
 
-} // end of namespace wave
+}  // end of namespace wave
```

Comment:
ah, I still think we can tune cpplint to our desire, for example to ignore the namespace warning you can create a `CPPLINT.cfg` config file with the following:

    filter=-readability/namespace

to disable the `// namespace wave` warning.